include/nlohmann/detail/input/input_adapters.hpp

```diff
@@ -418,7 +418,7 @@ typename container_input_adapter_factory_impl::container_input_adapter_factory<C
 }
 
 // specialization for std::string
-using string_input_adapter = decltype(input_adapter(std::declval<std::string>()));
+using string_input_adapter_type = decltype(input_adapter(std::declval<std::string>()));
```

Comment:
This was requested in the main pr due to the way we use it now right ?

Reply:
yes